example/spec/parallel/startSF/startSfSpec.js

```diff
@@ -181,31 +184,35 @@ describe('the sf-starter lambda function', () => {
     };
   });
 
-  describe('when provided a queue', () => {
+  describe('when linked to a queue', () => {
     const initialMessageCount = 30;
-    const testMessageLimit = 25;
 
-    let messagesConsumed;
     let passSfArn;
-    let qAttrParams;
     let queueName;
     let queueUrl;
+    let queueArn;
     let sfStarterName;
+    let executionPayload;
 
     beforeAll(async () => {
       sfStarterName = `${config.stackName}-sqs2sf`;
 
       queueName = `${testName}Queue`;
+      executionPayload = { test: testName };
 
       const { QueueUrl } = await sqs().createQueue({
-        QueueName: queueName
+        QueueName: queueName,
+        Attributes: {
```

Comment:
No issue with it, but why up the visibility timeout to 360?

Reply:
When trying to create an `EventSourceMapping`, the `VisibilityTimeout` of the queue has to be greater than the lambda's timeout. Amazon recommends 6x the lambda's timeout to allow the service to attempt retries, which I carried forward to the module's configuration, but that seemed kind of excessive so I capped the tests at 360 because it is 1 minute greater than 5 minutes, which tends to be the maximum timeout we set, and also about as long as I would like any single integration to run in the worst case. It doesn't correspond to the Lambda service maximum these days since that is now 900 seconds for a function, but 🤷‍♂.